src/main/java/org/influxdb/impl/InfluxDBResultMapper.java

```diff
@@ -172,13 +222,22 @@ void cacheMeasurementClass(final Class<?>... classVarAgrs) {
     }
   }
 
-  String getMeasurementName(final Class<?> clazz) {
+  public String getMeasurementName(final Class<?> clazz) {
     return ((Measurement) clazz.getAnnotation(Measurement.class)).name();
   }
 
+  public <T> ConcurrentMap<String, Field> getColNameAndFieldMap(final Class<T> clazz) {
```

Comment:
There is no need to have this method: you are not mocking it and now you are exposing an internal structure publicly to the external code, allowing it to be changed. **Please, revert this change.**